CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ These are the section headers that we use:
 - Added `POST /api/v1/me/datasets/{dataset_id}/records/search` endpoint ([#3068](https://github.com/argilla-io/argilla/pull/3068)).
 - Added new components in feedback task Question form: MultiLabel ([#3064](https://github.com/argilla-io/argilla/pull/3064)) and SingleLabel ([#3016](https://github.com/argilla-io/argilla/pull/3016)).
 - Added docstrings to the `pydantic.BaseModel`s defined at `argilla/client/feedback/schemas.py` ([#3137](https://github.com/argilla-io/argilla/pull/3137))
+- Added search component for feedback task ([#3138](https://github.com/argilla-io/argilla/pull/3138))
```

Comment:
please, include changes under the proper release (1.10.0) It have been included in 1.9.0 instead